packages/domain-picker/src/components/domain-suggestion-item/suggestion-item.tsx

```diff
@@ -23,15 +25,16 @@ import { sprintf } from '@wordpress/i18n';
 import InfoTooltip from '../info-tooltip';
 import WrappingComponent from './suggestion-item-wrapper';
 // TODO: remove when all needed core types are available
-/*#__PURE__*/ import '../types-patch';
+/*#__PURE__*/ import '../../types-patch';
 
-export const ITEM_TYPE_RADIO = 'radio';
-export const ITEM_TYPE_BUTTON = 'button';
-export const ITEM_TYPE_INDIVIDUAL_ITEM = 'individual-item';
+export const SUGGESTION_ITEM_TYPE_RADIO = 'radio';
+export const SUGGESTION_ITEM_TYPE_BUTTON = 'button';
+export const SUGGESTION_ITEM_TYPE_INDIVIDUAL = 'individual-item';
 export type SUGGESTION_ITEM_TYPE =
-	| typeof ITEM_TYPE_RADIO
-	| typeof ITEM_TYPE_BUTTON
-	| typeof ITEM_TYPE_INDIVIDUAL_ITEM;
+	| typeof SUGGESTION_ITEM_TYPE_RADIO
+	| typeof SUGGESTION_ITEM_TYPE_BUTTON
+	| typeof SUGGESTION_ITEM_TYPE_INDIVIDUAL;
+
```

Comment:
[nitpick/] This is scope creep since it doesn't have anything to do with folder restructuring.

Reply:
See https://github.com/Automattic/wp-calypso/pull/51235#discussion_r599720224